### PR TITLE
Added `Flagged`, a mod that adds animated flags.

### PR DESCRIPTION
**Pull request by NickStalburg**
It just adds animated flags using flag textures from Frozen Hell nations.

### CODE REVIEW

File: packs/descentfrozenhell.voodoo.kts [hunk 1]
```diff
@@ -210,10 +210,13 @@ root<Curse> {
             //+"opensolidstate" url "https://launcher.towerdevs.xyz/external/opensolidstate-1.0.jar"
         }
 
-        withTypeClass(Local::class).list {
+               withTypeClass(Local::class).list {
             +"thaumictinkerer" {
                 fileSrc = "thaumictinkerer-1.12.2-5.0-41d79cc.jar"
             }
+            +"flagged" {
+                fileSrc = "flagged-1.12.2-0.1.1.jar"
```
Comment (CitadelCore) [inline packs/descentfrozenhell.voodoo.kts:218]: How will the client fetch this file?

Comment (skyem123) [reply packs/descentfrozenhell.voodoo.kts:218]: It's in the local folder... https://github.com/CitadelCore/DescentPacks/pull/109/files#diff-48d3f6294f20e3342f722ca10175d5c0

File: packs/descentfrozenhell.voodoo.kts [hunk 2]
```diff
@@ -210,10 +210,13 @@ root<Curse> {
             //+"opensolidstate" url "https://launcher.towerdevs.xyz/external/opensolidstate-1.0.jar"
         }
 
-        withTypeClass(Local::class).list {
+               withTypeClass(Local::class).list {
```
Comment (skyem123) [inline packs/descentfrozenhell.voodoo.kts:213]: Indentation has been messed up...

Comment (NickStalburg) [reply packs/descentfrozenhell.voodoo.kts]: Fixed the mistake.

**Pull request merged**